src/Microsoft.TestPlatform.CommunicationUtilities/Resources/Resources.resx

```diff
@@ -118,10 +118,10 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="AbortedTestDiscovery" xml:space="preserve">
-    <value>The active Test Discovery was aborted.</value>
+    <value>The active Test Discovery was aborted. Due to {0}</value>
```

Comment:
It is possible that `{0}` is replaced with empty string for output. Is that expected? It gives an impression that something abnormal happened and the entire string is not printed.